### PR TITLE
Training 2023/01/30 PAT: Fixups

### PR DESCRIPTION
**Pull request by leogermond**
Major
- Base types: Add slide on numeric types conversion
- Records : Added very quick intro to limited (useful for classes without limited section)
- Subprograms: add example for default value
- Access: Rephrase access check rule, this version is probably less confusing

Minor (but impacting many files)
- Quiz: fix for comment that appeared in the slides, fix a typo `Integer = 123` -> `:=`
- Files preludes: added `|ge| |le| |gt| |lt|`
- Solar system labs: fix lab names

### CODE REVIEW

File: courses/fundamentals_of_ada/030_basic_types.rst [hunk 1]
```diff
@@ -843,6 +847,28 @@ Floating Point Type Attributes
...
 
+---------------------------
+Numeric Types Conversion
+---------------------------
+
+* Ada's integer and real are :dfn:`numeric`
+
+    - Holding a numeric value
+
+* Special rule: they can always be converted inbetween them
```
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/030_basic_types.rst:858]: "in-between" or "in between"
Or maybe better: "Can always convert between integer and real types"

File: courses/fundamentals_of_ada/030_basic_types.rst [hunk 2]
```diff
@@ -843,6 +847,28 @@ Floating Point Type Attributes
...
+
+    - Explicitely
```
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/030_basic_types.rst:860]: "Explicitly"

File: courses/fundamentals_of_ada/030_basic_types.rst [hunk 3]
```diff
@@ -843,6 +847,28 @@ Floating Point Type Attributes
...
+    - :ada:`Real` |rightarrow| :ada:`Integer` causes **rounding**
```
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/030_basic_types.rst:861]: Do we want want to mention rounding occurs away from 0? (e.g. 1.5 => 2, -1.5 => -2)
Or is that to much info?

File: courses/fundamentals_of_ada/030_basic_types.rst (hunk 1 above)
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/030_basic_types.rst:858]: "in-between" or "in between"
Or maybe better: "Can always convert between integer and real types"

File: courses/fundamentals_of_ada/030_basic_types.rst (hunk 2 above)
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/030_basic_types.rst:860]: "Explicitly"

File: courses/fundamentals_of_ada/070_subprograms.rst [hunk 4]
```diff
@@ -381,14 +385,20 @@ Actual Parameters Respect Constraints
      Foo (Q); -- runtime error if Q <= 0
      Foo (P);
 
------------------
-Parameter Modes
------------------
+---------------------------
+Parameter and Return Mode
```
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/070_subprograms.rst:389]: No reference to "Return" on this page
If you add one, it should be "Parameter Modes and Return"

File: courses/fundamentals_of_ada/140_access_types.rst [hunk 5]
```diff
@@ -454,7 +458,10 @@ Introduction to Accessibility Checks (1/2)
                -- Nested subprogram, enclosing + 1, here 2
                O2 : aliased Integer;
 
-* Access **types** can only access objects that are at **same or lower depth**
+* Objects can be access by access **types** that are at **same depth or deeper**
```
Comment (michael-frank-62) [inline courses/fundamentals_of_ada/140_access_types.rst:461]: should be "accessed by", but better would be "referenced by" or "pointed to by"

**Review by michael-frank-62 — CHANGES_REQUESTED** [review]

File: courses/fundamentals_of_ada/030_basic_types.rst (hunk 3 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/030_basic_types.rst:861]: Yes this is important, I added it to the `Float'Rounding` description (previous slide)

File: courses/fundamentals_of_ada/070_subprograms.rst (hunk 4 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/070_subprograms.rst:389]: The last bullet is referencing return

File: courses/fundamentals_of_ada/140_access_types.rst (hunk 5 above)
Comment (leogermond) [reply courses/fundamentals_of_ada/140_access_types.rst:461]: agreed

**Review by michael-frank-62 — APPROVED** [review]

**Review by frank-at-adacore — APPROVED** [review]

**Pull request merged**